src/crimson/os/seastore/ordering_handle.h

```diff
@@ -0,0 +1,69 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
nit, might want to s/t/nil/ in new source files.

Reply:
Why?  I haven't seen anything about this.